definitions/infra-host/definition.yml

```diff
@@ -13,6 +13,36 @@ goldenTags:
 - azure.resourceGroup
 - gcp.zone
 - gcp.projectId
+# opentelemetry semantic conventions for cloud instances
+- cloud.provider
+- cloud.account.id
+- cloud.region
+- cloud.availability_zone
+- cloud.platform
+synthesis:
+  rules:
+    # opentelemetry host data from opentelemetry-collector
+    - identifier: host.id
```

Comment:
infra|na hosts use a privately generated domainId, right? Is this attr value the same?

Reply:
ah, i think i get what's going on - this is telemetry from the nr agent, but you're somehow flagging it with additional instrumentation.provider info, right?

Reply:
this is telemetry coming from the opentelemetry collector. I'm trying to investigate we can classify these as infra hosts (as opposed to ext-hosts) in order to view all your hosts together, regardless of their source.